spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe QueueItem do
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+
+  describe "#video_title" do
```

Comment:
For this you can even use shoulda matchers's `delegate_method` - check their docs for this - makes your tests a little bit more succinct.